mqclient/queue.py

```diff
@@ -421,3 +426,17 @@ def get_message_callback(msg: Message) -> Message:
             )
 
         return self.msg.data
+
+    @wtt.spanned(
+        these=[
+            "self.queue._backend",
+            "self.queue._address",
+            "self.queue._name",
+            "self.queue._prefetch",
+            "self.queue.timeout",
+        ],
+    )
+    async def nack_current(self) -> None:
+        """Manually nack the current (most recently yielded) message."""
+        # pylint:disable=protected-access
+        await self.queue._safe_nack(self._sub, self.msg)
```

Comment:
We want the exception when the user calls this on an already-ACK'd right?

Reply:
`_safe_nack()` has a guardrail for this. In practice, the generator acks at the end of the loop plus the user can't manually ack, so this can't happen.